community/server/src/main/java/org/neo4j/server/configuration/ConfigWrappingConfiguration.java

```diff
@@ -60,31 +60,21 @@ public boolean containsKey( String key )
     public Object getProperty( String key )
     {
         Setting<?> setting = getSettingForKey( key );
-        if( setting == null )
-        {
-            return null;
-        }
-        else
-        {
-            return config.get( setting );
-        }
+        return setting == null ? config.getParams().get( key ) : config.get( setting );
     }
 
     @Override
     public Iterator<String> getKeys()
     {
-        // get all the properties keys
-        Set<String> staticKeys = getRegisteredSettings().keySet();
-        // only keep the properties which have been assigned default values or user-defined values
-        Set<String> notNullKeys = new HashSet<>();
-        for ( String key : staticKeys )
+        Set<String> propertyKeys = new HashSet<>(config.getParams().keySet());
```

Comment:
How about also keep the comments in case we forgot to filter properties without a default value in the future? Besides, plz use the default code style, you are missing spaces here :)